src/main/java/emissary/spi/SPILoader.java

```diff
@@ -0,0 +1,28 @@
+package emissary.spi;
+
+import java.util.ServiceLoader;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Load SPI implementations to support initialization of the Emissary server.
+ * 
+ * <pre>
+ * - Loading is delegated to the {@link java.util.ServiceLoader}.
+ * - Configured via src/main/resources/META-INF/services/emissary.spi.InitializationProvider
+ * </pre>
+ */
+public class SPILoader {
+
+    private static final Logger logger = LoggerFactory.getLogger(SPILoader.class);
+
+    public static void load() {
```

Comment:
Is there any risk in this loading not being synchronized? I don't think so, but it's possible a future provider could perform work that isn't safe for concurrent execution.